packages/components/table/examples/TableWithSortingPropsExample.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { Badge, Table, TableCellSorting } from '@contentful/f36-components';
+
+export default function TableWithSorting() {
+  const contentTypes = [
+    {
+      id: '1',
+      name: 'Category',
+      description:
+        'Categories can be applied to Courses and Lessons. Assigning Multiple categories is also possible.',
+      updatedAt: 'Nov 15, 2021',
+      status: 'published',
+    },
+    {
+      id: '2',
+      updatedAt: 'Nov 15, 2021',
+      status: 'draft',
+    },
+    {
+      id: '3',
+      name: 'Layout',
+      description:
+        'A page consisting of freely configurable and rearrangeable content modules.',
+      updatedAt: 'Nov 15, 2021',
+      status: 'published',
+    },
+  ];
+
+  const [sorting, setSorting] = useState(undefined);
```

Comment:
Maybe it's better to provide some sorting (e.g. `TableCellSorting.Ascending`) by default because it was confusing that I have to click first on the Column heading to be able to see the sorting arrows in the example.

Reply:
I actually don't think I agree here, it's seems quite common when I think of column sorting that the default is "unsorted". What we could do to fully support this is actually that on the third click we cycle back to this unsorted state.

Reply:
But I made the sorting a bit more understandable. I think.

Reply:
The sortable example from Shopify is actually sorted by default by `Net sales` column, so we can see the sorting arrows when we open the page. That's what I meant we can do with our example as well. 
I don't think that cycling to the unsorted state is a good idea since that's not how sorting in real-world tables works.

But not a blocker.

Reply:
And the example code is a bit hard to read and understand since it has a lot of ternary operators. I'd prefer to hide the complexity that users don't need to get an idea of how to add sorting to our table. Shopify did a good job on it by moving sorting logic to a separate function.
<img width="703" alt="image" src="https://user-images.githubusercontent.com/10744462/227875668-83459302-c2c1-45a5-8541-431392bd3353.png">

Also no a blocker.